harmony_regridding_service/regridding_service.py

```diff
@@ -78,9 +79,14 @@ def regrid(
         )
 
         vars_to_process = var_info.get_all_variables()
+        unresampled_vars = unresampled_variables(var_info)
+        grid_mappings = input_grid_mappings(source_ds, vars_to_process)
+
+        logger.info(f'dropping grid_mappings: {grid_mappings}')
+        vars_to_process -= grid_mappings
```

Comment:
This is what changed.  we used to copy all the vars over from the input file, but now we drop any that were references as a grid_mapping so that when we update the vars with new grid_mappings we don't have unreferrenced and incorrect variables on the output file.